### PR TITLE
qutebrowser: bump version

### PR DESCRIPTION
**Pull request by kenmays**
UPDATE: QuteBrowser 1.8.1, Tested on hrev53511.

### CODE REVIEW

**Comment by kenmays** [comment]
Updated. Good to merge.

**Comment by extrowerk** [comment]
Builds, packages fine on 64 bit

**Comment by Begasus** [comment]
ninja_python3 can't be installed due to missing package for markupsafe_python3 for x86_gcc2

**Comment by kenmays** [comment]
qutebrowser-1 7 0-haiku_cocobeaan
Qutebrowser 1.7.0 on x86. Python 3.6 downloads MarkupSafe 1.1.1 w/Jinja.

**Comment by Begasus** [comment]
Looks good, just how did you get MarkupSafe 1.1.1 installed? Couldn't find a recipe for it in Haikuports

**Comment by kenmays** [comment]
Src imports from the Net if not in the repo, if a specific version is defined.

**Comment by Begasus** [comment]
Maybe if you build from Terminal? haikuporter will pull the package from the depot, anyway, created PR's to bump both jinja and markupsafe to bring them back on par :)
Mind you jinja3 will point to python3.7, if you need python3.6 use jinja36 then (same for markupsafe36)

**Comment by Begasus** [comment]
markupsafe and jinja have been merged, with a small change to the recipe (as mentioned in my earlier comment) it builds and installs fine, only thing is that I have to enable QTWebkit backend to start it up and got this warning:
QuteBrowser
Other then that it starts up without the warning (even could watch youtube movie with it) :+1:

File: www-client/qutebrowser/qutebrowser-1.7.0.recipe [hunk 1]
```diff
@@ -27,7 +27,7 @@ REQUIRES="
 	pypeg2_python3
 	pyqt_python3
 	pyyaml_python3
-	setuptools_python3
+	setuptools_python36
```
Comment (Begasus) [inline www-client/qutebrowser/qutebrowser-1.7.0.recipe]: This is also needed for jinja_python3 vs jinja_python36 :)

**Comment by Begasus** [comment]
With my build the binaries go into $prefix/bin instead of $binDir, so a few changes needed:
1 missing: cmd:pygmentize (in PROVIDES)
2 change $binDir to $prefix/bin (doesn't change anything on x86_64): mv $prefix/bin/qutebrowser $appsDir/QuteBrowser (in INSTALL)

File: www-client/qutebrowser/qutebrowser-1.7.0.recipe (hunk 1 above)
Comment (kenmays) [reply www-client/qutebrowser/qutebrowser-1.7.0.recipe]: Resolved

File: www-client/qutebrowser/qutebrowser-1.8.1.recipe [hunk 2]
```diff
@@ -1,13 +1,13 @@
 SUMMARY="A VIM-like web browser"
 DESCRIPTION="QuteBrowser is a keyboard-focused browser with a minimal GUI. \
-It’s based on Python and PyQt5 and free software, licensed under the GPL. \
+Itâ€™s based on Python and PyQt5 and free software, licensed under the GPL. \
```
Comment (diversys) [inline www-client/qutebrowser/qutebrowser-1.8.1.recipe]: Typo?

Comment (kenmays) [reply www-client/qutebrowser/qutebrowser-1.8.1.recipe]: fixed

**Pull request closed without merging**